org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend

```diff
@@ -113,6 +113,10 @@ class WollokRepl {
 		println(obj?.toString.returnStyle)
 	}
 
+	def dispatch doPrintReturnValue(WollokObject wo) {
+		println(wo?.call("printString").toString.returnStyle)
+	}
+
 	def dispatch doPrintReturnValue(String obj) {
```

Comment:
As Javi told me, Repl used dispatched method for Strings when Wollok used Java objects (previous to big Refactor). If you think it is a good choice I can remove def dispatch doPrintReturnValue(String obj)